packages/calypso-e2e/src/data-helper.ts

```diff
@@ -121,7 +97,7 @@ export function getUsername( { prefix = '' }: { prefix?: string } = {} ): string
  * @returns {string|null} If valid date format string is supplied, string is returned. Otherwise, null.
  */
 export function getDateString( format: DateFormat ): string | null {
-	if ( format === 'ISO' ) {
+	if ( format === 'ISO-8601' ) {
```

Comment:
I added the `-8601` postfix to make the format specific.

Not that it really matters; this method is [not used anywhere](https://github.com/Automattic/wp-calypso/search?q=getDateString).